base/sparse/linalg.jl

```diff
@@ -510,7 +510,8 @@ end
 function normestinv{T}(A::SparseMatrixCSC{T}, t::Integer = min(2,maximum(size(A))))
     maxiter = 5
     # Check the input
-    n = max(size(A)...)
+    chksquare(A)
```

Comment:
`chksquare` returns the size so you can just do `n=chksquare(A)` and remove the next line.